src/solvers/dgmulti/flux_differencing_gauss_sbp.jl

```diff
@@ -25,15 +25,262 @@ function tensor_product_quadrature(element_type::Hex, r1D, w1D)
   return rq, sq, tq, wq
 end
 
-# Todo: DGMulti. Decide if we should add GaussSBP on triangles.
+# type parameters for TensorProductFaceOperator
+abstract type AbstractGaussOperator end
+struct Interpolation <: AbstractGaussOperator end
+struct Projection  <: AbstractGaussOperator end
+
+"""
+  struct TensorProductGaussFaceOperator{Tmat, Ti}
+
+Data for performing tensor product interpolation from volume nodes to face nodes.
+"""
```

Comment:
Do you consider this to be part of the API or not? If it's API, I would prefer to also list some constructor with arguments. If it's not API, we might turn this docstring into a plain comment to clarify that it's not part of our API.

Reply:
I don't think it should be part of the API - this should just be internal. I'll add that to the docstring